man/chpldoc.txt

```diff
@@ -30,8 +28,14 @@ DESCRIPTION
                     sequence used to distinguish a documentation comment
                     from a normal one (defaults to '/*' if unspecified).
 
+  --save-sphinx <directory>  Save generated Sphinx project in directory.
+
   --text-only       Specify that documents should use a text output file
-                    format rather than HTML.
```

Comment:
Or maybe matching some of the wording in the "DESCRIPTION" section

```
Generate text based documentation instead of HTML. Takes precedence over --[no-]html
```

and for --[no-]html:

```
[Don't] generate HTML based documentation (on by default)
```

what do you think?